### PR TITLE
Contract should not store empty alias in usedAliases and should not allow new user to set empty alias

### PR DESCRIPTION
**Pull request by igorline**

### CODE REVIEW

File: packages/kyodo-contracts/contracts/MembersV2.sol [hunk 1]
```diff
@@ -10,12 +10,23 @@ contract MembersV2 is MembersV1 {
...
+      require(!prevValue.toSlice().empty());
+    }
+
     int prevIndex = getNickNameIndex(prevValue);
+
     whitelist[msg.sender].alias = _value;
 
     // update used aliases
-    usedAliases.push(_value);
+    if (!isAliasEmpty) {
```
Comment (jsidorenko) [inline packages/kyodo-contracts/contracts/MembersV2.sol]: This check is obsolete as the `require(!prevValue.toSlice().empty());` block above would terminate the execution

File: packages/kyodo-contracts/contracts/MembersV2.sol [hunk 2]
```diff
@@ -10,12 +10,23 @@ contract MembersV2 is MembersV1 {
     external
   {
     require(nickNameNotExist(_value));
+    
+    bool isAliasEmpty = _value.toSlice().empty();
+
     string storage prevValue = whitelist[msg.sender].alias;
+
+    if (isAliasEmpty) {
+      require(!prevValue.toSlice().empty());
```
Comment (jsidorenko) [inline packages/kyodo-contracts/contracts/MembersV2.sol]: Shall we start writing an error message as a second param in `require` function?

Comment (igorline) [reply packages/kyodo-contracts/contracts/MembersV2.sol]: Added in the latest commit

File: packages/kyodo-contracts/contracts/MembersV2.sol (hunk 1 above)
Comment (igorline) [reply packages/kyodo-contracts/contracts/MembersV2.sol]: Require block would terminate only in case alias is empty and previous value was empty

Comment (jsidorenko) [reply packages/kyodo-contracts/contracts/MembersV2.sol]: You're right!

**Pull request merged**